### ISSUE TITLE
Add pre-requisites paragraph

### ISSUE THREAD

**Issue by boillodmanuel** [smallhadroncollider/brok#15]
Currently, on a debian stretch, I got this error:
```./brok: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by ./brok)```

**Comment by smallhadroncollider** [comment]
Do you know which package needs to be installed to fix this?

**Comment by boillodmanuel** [comment]
No, I don't. It's why I move to a docker installation (see #16)

**Comment by smallhadroncollider** [comment]
Fair enough! Hopefully someone else will. Or I might get a chance to test it at some point. Sorry, super busy taking my work remote for the last few months. Hopefully will calm down soon.
